#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_update_for_gnomedesktop_SONAME.dpatch by Christopher James Halse Rogers <raof@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Update libgnomedesktop2.20-cil DLL map for new SONAME in Lucid.

@DPATCH@
diff -urNad gnome-desktop-sharp2-2.26.0~/gnomedesktop/gnomedesktop-sharp.dll.config.in gnome-desktop-sharp2-2.26.0/gnomedesktop/gnomedesktop-sharp.dll.config.in
--- gnome-desktop-sharp2-2.26.0~/gnomedesktop/gnomedesktop-sharp.dll.config.in	2009-02-24 05:01:27.000000000 +1100
+++ gnome-desktop-sharp2-2.26.0/gnomedesktop/gnomedesktop-sharp.dll.config.in	2010-02-06 15:35:56.512813842 +1100
@@ -3,5 +3,5 @@
   <dllmap dll="libgobject-2.0-0.dll" target="libgobject-2.0@LIB_PREFIX@.0@LIB_SUFFIX@"/>
   <dllmap dll="libgthread-2.0-0.dll" target="libgthread-2.0@LIB_PREFIX@.0@LIB_SUFFIX@"/>
   <dllmap dll="libgtk-win32-2.0-0.dll" target="libgtk-x11-2.0@LIB_PREFIX@.0@LIB_SUFFIX@"/>
-  <dllmap dll="gnome-desktop-2" target="libgnome-desktop-2@LIB_PREFIX@.11@LIB_SUFFIX@"/>
+  <dllmap dll="gnome-desktop-2" target="libgnome-desktop-2@LIB_PREFIX@.17@LIB_SUFFIX@"/>
 </configuration>
